docs/manual/contributing/_index.de.md

```diff
@@ -0,0 +1,257 @@
+---
+title: "Zur Dokumentation beitragen"
+description: "Hier erfährst du wie du zur Dokumentation beitragen kannst."
+url: "beitragen"
+aliases:
+    - /de/beitragen/
+weight: 300
+---
+
+## Vor deinem ersten Beitrag
+
+Bevor du an der Dokumentation mit schreiben kannst, musst du noch folgendes tun:
+
+- Erstelle ein kostenloses Konto bei [GitHub](https://github.com/join). [GitHub](https://de.wikipedia.org/wiki/GitHub) 
+ist eine Versionsverwaltung für Software-Projekte und beheimatet unter anderem die 
+[Contao-Dokumentation](https://github.com/contao/docs/).
+- Die Dokumentation wird in [Markdown](https://de.wikipedia.org/wiki/Markdown) geschrieben, mache dich mit der 
+[Syntax](https://docs.contao.org/manual/de/artikelverwaltung/inhaltselemente/#syntax) dieser Auszeichnungssprache 
+ertraut.
+
+
+## Kleine Fehler korrigieren
+
+Wenn du während dem Lesen der Dokumentation einen Tippfehler entdeckst oder etwas umformulieren möchtest, ist es am 
+einfachsten, dies direkt über GitHub zutun.
+
+
+### Seite bearbeiten und Fork des Repositories erstellen
+
+Klicke auf den Link »Diese Seite bearbeiten« in der oberen rechten Ecke und du wirst zu GitHub weitergeleitet:
+
+![Diese Seite bearbeiten](/de/contributing/images/de/diese-seite-bearbeiten.png?classes=shadow)
+
+Bei deinem ersten Beitrag, wirst du gebeten einen [Fork](https://de.wikipedia.org/wiki/Abspaltung_(Softwareentwicklung)) 
+vom [Repository](https://de.wikipedia.org/wiki/Repository) »contao/docs« zu erstellen, klicke auf »Fork this repository«.
+
+![Fork this repository](/de/contributing/images/de/fork-this-repository.png?classes=shadow)
+
+
+### Inhalt bearbeiten
+
+Bearbeite den Inhalt, beschreibe deine Änderungen und klicke auf die Schaltfläche »Propose file change«.
+
+![Bearbeite den Inhalt](/de/contributing/images/de/inhalt-bearbeiten.png?classes=shadow)
+
+
+### Branch und Commit erzeugen
+
+GitHub wird nun einen Branch und einen Commit für deine Änderungen erzeugen ausserdem wird eine Vorschau deiner 
+Änderungen angezeigt:
+
+![Branch und Commit erzeugen](/de/contributing/images/de/branch-und-commit-erzeugen.png?classes=shadow)
+
+Wenn alles korrekt ist, klicke auf die Schaltfläche »Create pull request«.
+
+
+### Pull-Request erzeugen
+
+Auf der nächsten Seite, kannst du, falls nötig, noch letzte Anpassungen an deinen Pull-Request vornehmen. 
```

Comment:
Vielleicht allgemein kurz erwähnen, was ein "Pull-Request" ist und warum er so heisst? Als Info-Box oder so?
So ähnlich vielleicht:

"Im verteilten Versionierungssystem Git und somit auch auf GitHub werden Vorschläge als sog. "Pull-Requests" erstellt. Da du keinen Berechtigungen hast, direkt Änderungen im offiziellen Repository zu vollziehen ("to commit") stellst eine Anfrage (engl. "Request") an die Berechtigten des Repositories, deine Änderungen in das offizielle Repository zu "ziehen" (engl. "to pull")."